src/web/components/form/ErrorMarker.jsx

```diff
@@ -0,0 +1,29 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import React from 'react';
+
+import styled from 'styled-components';
+
+import {styledExcludeProps} from 'web/utils/styledConfig';
+import Theme from 'web/utils/theme';
+
+const StyledMarker = styledExcludeProps(styled.div, ['isVisible'])`
```

Comment:
Alternatively, to  `styledExcludeProps` it should be possible to do `$prop`

Reply:
I cleaned up the component a bit instead of leaving it as it is.